packages/react-dom/src/__tests__/ReactDOMServerIntegrationContext-test.js

```diff
@@ -108,7 +108,7 @@ describe('ReactDOMServerIntegration', () => {
     itRenders('stateless child without context', async render => {
       function StatelessChildWithoutContext(props, context) {
         // this should render blank; context isn't passed to this component.
-        return <div>{context.text}</div>;
+        return <div>{context === undefined ? null : context.text}</div>;
```

Comment:
This looks weird. Should we just put an `expect` here that asserts `context` is undefined?